### ISSUE TITLE
Setup Wandb.ai: cannot find group

### ISSUE THREAD

**Issue by jcohenadad** [ivadomed/ivadomed#1212]
I'm trying to setup Wandb.ai for training.

I've created a 'Personnal' account, then a project name, but I am confused about the group name. In the doc it says:
> On the WandB Dashboard, the groups can be found on clicking the Projects tab on the left. Default: my_group.

However I don't see this in my Wandb page:

![image](https://user-images.githubusercontent.com/2482071/207379323-a95c5fe5-83c4-423b-86bb-2e5058860d18.png)

**Comment by jcohenadad** [comment]
I can still use Wandb without setting the 'group', but I'd suggest to clarify the doc.

**Comment by kanishk16** [comment]
> On the WandB Dashboard, the groups can be found on clicking the Projects tab on the left. Default: my_group.


![image](https://user-images.githubusercontent.com/36276423/207579308-403d8874-92ae-425e-bd08-d41ee6eda716.png)

On clicking the Projects Tab, 

![image](https://user-images.githubusercontent.com/36276423/207590262-f16c44e8-014b-4df6-ad3b-347147a2a240.png)

**Comment by jcohenadad** [comment]
Thank you @kanishk16 . So, from a 'user' perspective, how should we do it, given that when starting a new project, groups are not defined yet. And BTW, how important is it to set up groups in our usage of wandb? what are they used for? if not useful i'd suggest to say something like: "you can leave this field as is"

**Comment by kanishk16** [comment]
> So, from a 'user' perspective, how should we do it, given that when starting a new project, groups are not defined yet.

@jcohenadad The following is a great recommendation from our docs as a [note](https://ivadomed.org/configuration_file.html#run-name):

> We recommend defining the project/group/run names such that hierarchy is easily understandable. For instance, project_name could be the name of the dataset or the problem you are working (i.e. brain tumor segmentation/spinal cord lesion segmentation etc.), the group_name could be the various models you are willing to train, and the run_name could be the various experiments within a particular model (i.e. typically with different hyperparameters). 

I guess the above should clear out most of the queries. BTW feel free to omit the group_name key for now which defaults to `my_group`. 

The real benefit is when you'd like to compare different runs of different models. Let's assume the following scenario:
```
├── Model A <- group name
│   ├── run1 (with rotation)
│   └── run2 (without rotation)
└── Model B <- group name
    ├── run1 (with rotation)
    └── run2 (without rotation)
```
and we'd like to compare `Model A`: `run1` vs `run2`:

![image](https://user-images.githubusercontent.com/36276423/207656386-c9151d7b-6fe0-4a6d-be2f-f2ed97de3064.png)

and then later you'd like to compare only `Model A + run1` vs `Model B + run1`. This can be done by hiding or unhiding (clicking on the eye) as below:

![image](https://user-images.githubusercontent.com/36276423/207657358-e5f4e8e0-229d-4834-bdb8-eedadbf936fe.png)

**Comment by jcohenadad** [comment]
@kanishk16 I agree the existing note is informative enough. I had read it, but somehow it didn't make sense to me at that time. It does now. So I guess we can close this issue.
